dbt/include/global_project/macros/materializations/seed/seed.sql

```diff
@@ -0,0 +1,126 @@
+
+{% macro create_csv_table(model) -%}
+  {{ adapter_macro('create_csv_table', model) }}
+{%- endmacro %}
+
+{% macro reset_csv_table(model, full_refresh, existing) -%}
+  {{ adapter_macro('reset_csv_table', model, full_refresh, existing) }}
+{%- endmacro %}
+
+{% macro load_csv_rows(model) -%}
+  {{ adapter_macro('load_csv_rows', model) }}
+{%- endmacro %}
+
+{% macro default__create_csv_table(model) %}
+  {%- set agate_table = model['agate_table'] -%}
+  {%- set column_override = model['config'].get('column_types', {}) -%}
+
+  {% set sql %}
+    create table {{ model['schema'] }}.{{ model['name'] }} (
```

Comment:
you want `{{ api.Relation.create_from_node(model) }}` here -- this is causing the tests to break

Reply:
ok i am confused -- actually what you need to do to make the tests pass in development is quote the schema and identifier here